domains/blaine.json

```
@@ -0,0 +1,13 @@
+{
+    "description": "The face of Blaine, on the web",
+    "repo": "https://github.com/Blakeinstein/Blakeinstein.github.io",
+    "owner": {
+      "username": "Blakeinstein",
+      "email": "rishi1998@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://blakeinstein.github.io",
```

CNAME has to be a host. Full URLs dont work
```suggestion
      "CNAME": "blakeinstein.github.io",
```